packages/camera/camera/test/camera_test.dart

```diff
@@ -90,6 +90,10 @@ void main() {
 
       expect(camList, equals(mockAvailableCameras));
     });
+
+    test('CameraLensType is exported and usable', () {
```

Comment:
This doesn't actually test the fix; this file imports `camera_platform_interface.dart`, so it would compile without the export fix.

Generally we test exports with a new test file that's just for that, and which only imports the current package's main file. See https://github.com/flutter/packages/blob/main/packages/webview_flutter/webview_flutter/test/webview_flutter_export_test.dart for an example.